lib/blob/blob_bs_dev.c

```diff
@@ -183,9 +184,10 @@ blob_bs_is_zeroes(struct spdk_bs_dev *dev, uint64_t lba, uint64_t lba_count)
 	}
 
 	assert(blob->back_bs_dev != NULL);
-	return blob->back_bs_dev->is_zeroes(blob->back_bs_dev,
-					    bs_io_unit_to_back_dev_lba(blob, lba),
-					    bs_io_unit_to_back_dev_lba(blob, lba_count));
+	is_valid_range = blob->back_bs_dev->is_range_valid(blob->back_bs_dev, lba, lba_count);
```

Comment:
IIRC somewhere on your previous patch there a place where we check for range valid and zeroes separately. Will this break that logic?

Reply:
> IIRC somewhere on your previous patch there a place where we check for range valid and zeroes separately. Will this break that logic?

In the eventual patch, the checks are happening together everywhere. I don't see this causing trouble there. It's a nesting on that earlier check.